app/web/src/features/profile/edit/styles.ts

```diff
@@ -0,0 +1,81 @@
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  avatar: {
+    width: 120,
+    height: 120,
+  },
+  topFormContainer: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    [theme.breakpoints.up("md")]: {
+      flexDirection: "row",
+      margin: theme.spacing(1, 10),
+    },
+    "& .MuiTextField-root": {
+      width: "100%",
+    },
+  },
+  buttonContainer: {
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    width: "100%",
+    display: "flex",
+    zIndex: 105,
```

Comment:
Can this be done without arbitrary `z-index` values? I'm a bit wary of the [z-index inflation problem](https://www.joshwcomeau.com/css/stacking-contexts/#debugging-stacking-context-issues) briefly referenced here (the blog post is very good at talking about how to deal with `z-index` in general!) and this leading to more hacky CSS for us in the future

Reply:
I'm not sure it's possible because `tui-editor` sets a z-index which necessitates more of them....

Reply:
Exactly the tui-editor was the problem why I needed to use the z-index in this case. I don't think there is an easy fix for this

Reply:
Ahh yeah that's annoying :/ let's leave it then